### ISSUE TITLE
workspace needs date modified in addition to date created

### ISSUE THREAD

**Issue by hyoo** [PATRIC3/patric3_website#503]
it would be more useful if we can see & sort by date modified. Currently it shows date created and it would not display meaningful info, once folders are created a while ago.

**Comment by dmachi** [comment]
I don't believe this data currently exists in the workspace service.  I know there was some brief discussion of it when we travelled to ANL, but I don't know that anything was decided.  We'll need that data before we can show anything useful.

**Comment by mshukla1** [comment]
@nconrad, can you check if it is possible to add date modified as one of the columns in workspace file listing? 

If it is not supported by the current workspace service, lets close the ticket as won't fix. 

-Maulik

<img width="1128" alt="screen shot 2017-09-17 at 11 14 01 am" src="https://user-images.githubusercontent.com/1236463/30522655-5aca784e-9b99-11e7-997f-c993576edefd.png">

**Comment by nconrad** [comment]
@mshukla1, it's not possible because the ws service only provides the creation time, unfortunately.
